### PR TITLE
enhance: revert order of parameters for better readability

### PR DESCRIPTION
**Pull request by Beraliv**
The discussion about it is still open
https://github.com/ramda/ramda/issues/1497

### CODE REVIEW

**Comment by Beraliv** [comment]
As a solution it can be replace as following:

```js
const isLessThan5 = lt(5)
isLessThan5(4)  // true
lt(5, 6)                 // true
```

**Comment by Beraliv** [comment]
However we don't implement lodash, which has everything in one function
That's why it's possible to implement different functions with inverted order of parameters

**Comment by Beraliv** [comment]
Proposal should be applied to multiple lines
@BabkinAleksandr

File: lib/gte/gte.test.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import gte from '.'
 
-test('checks if 1st arg is greater or equals to 2nd', () => {
-  expect(gte(1)(2)).toBeFalsy()
-  expect(gte(2)(2)).toBeTruthy()
-  expect(gte(3)(2)).toBeTruthy()
+it('checks if value is greater than or equals to another one', () => {
+  const isGreaterOrEquals2 = gte(2)
+
+  expect(isGreaterThanOrEqualsTo2(5)).toBeTruthy()
```
Comment (BabkinAleksandr) [inline lib/gte/gte.test.js]:
```suggestion
  expect(isGreaterOrEquals2(5)).toBeTruthy()
```

File: lib/gte/gte.test.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+  expect(isGreaterThanOrEqualsTo2(2)).toBeTruthy()
```
Comment (BabkinAleksandr) [inline lib/gte/gte.test.js]:
```suggestion
  expect(isGreaterOrEquals2(2)).toBeTruthy()
```

File: lib/gte/gte.test.js [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+  expect(isGreaterThanOrEqualsTo2(0)).toBeFalsy()
```
Comment (BabkinAleksandr) [inline lib/gte/gte.test.js]:
```suggestion
  expect(isGreaterOrEquals2(0)).toBeFalsy()
```

File: lib/lte/lte.test.js [hunk 4]
```diff
@@ -1,7 +1,9 @@
 import lte from '.'
 
-test('checks if 1st arg is less or equals to 2nd', () => {
-  expect(lte(1)(2)).toBeTruthy()
-  expect(lte(2)(2)).toBeTruthy()
-  expect(lte(3)(2)).toBeFalsy()
+it('checks if value is less than or equals to another one', () => {
+  const isLessOrEquals2 = lte(2)
+
+  expect(isLessThanOrEqualsTo2(5)).toBeFalsy()
```
Comment (BabkinAleksandr) [inline lib/lte/lte.test.js]:
```suggestion
  expect(isLessOrEquals2(5)).toBeFalsy()
```

File: lib/lte/lte.test.js [hunk 5]
```diff
@@ -1,7 +1,9 @@
...
+  expect(isLessThanOrEqualsTo2(2)).toBeTruthy()
```
Comment (BabkinAleksandr) [inline lib/lte/lte.test.js]:
```suggestion
  expect(isLessOrEquals2(2)).toBeTruthy()
```

File: lib/lte/lte.test.js [hunk 6]
```diff
@@ -1,7 +1,9 @@
...
+  expect(isLessThanOrEqualsTo2(0)).toBeTruthy()
```
Comment (BabkinAleksandr) [inline lib/lte/lte.test.js]:
```suggestion
  expect(isLessOrEquals2(0)).toBeTruthy()
```

**Comment by Beraliv** [comment]
This is useless
It mixes everything up

**Pull request closed without merging**